haskell/README.md

```diff
@@ -6,3 +6,14 @@ SPDX-License-Identifier: MIT
 # Stablecoin Haskell
 
 This folder contains a Haskell library to interact with the stablecoin contract (basically Haskell bindings) and tests for it.
+
+## Build Instructions
+
+You need [Stack](http://haskellstack.org/) to build this package.
+Run `stack build` to build the library.
+
+## Tests
+
+Tests require the stablecoin contract to be in `test/resources/stablecoin.tz`.
+After you put it there you can do `stack test` to run tests.
```

Comment:
`stack test` should be `make test` right? because `stack test` does not build the contract?

Reply:
But I am saying "After you put it there". After you put it there you can indeed just run `stack test`. And the next line says that there is a `Makefile` that automates this process. I can add that one can run `make test`.